packages/angular_devkit/core/src/virtual-fs/path.ts

```diff
@@ -241,3 +241,25 @@ export function normalize(path: string): Path {
     return p.join(NormalizedSep) as Path;
   }
 }
+
+
+// Platform-specific paths.
+export type WindowsPath = string & {
+  __PRIVATE_DEVKIT_WINDOWS_PATH: void;
+};
+export type PosixPath = string & {
+  __PRIVATE_DEVKIT_WINDOWS_PATH: void;
```

Comment:
This should be `__PRIVATE_DEVKIT_POSIX_PATH` I think,

Reply:
Fixed thanks.